Homework_5/hw5_pandas_and_plots.py

```diff
@@ -0,0 +1,280 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[1]:
+
+
+import pandas as pd
+import numpy as np
+import matplotlib.pyplot as plt
+import seaborn as sns
+from matplotlib.patches import ConnectionPatch
+import matplotlib.colors as mcolors
+
+
+def read_gff(gff_file):
+    '''Function read_gff() reads either gziped or not .gff file and transforms it into pandas.DataFrame
+    
+    Parameters:
+    gff_file (str): path to .gff input file
+    
+    Returns: pandas.DataFrame 
+    '''
+    
+    df_gff = pd.read_table(gff_file, sep='\t', skiprows=1, names=['chromosome', 'source', 'type', 'start', 
```

Comment:
Твой код учитывает только случай, где комментарий занимает только первую строку. Однако чаще всего в файлах данного формата могут быть комментарии на много строк, их нужно учитывать. Благо в пандасе это делается супер просто
```suggestion
    df_gff = pd.read_table(gff_file, sep='\t', comment="#", names=['chromosome', 'source', 'type', 'start', 
```